.circleci/config.yml

```diff
@@ -74,6 +76,19 @@ jobs:
             git remote add ro https://github.com/fairwindsops/charts
             git fetch ro master
             ct lint --config scripts/ct.yaml --chart-yaml-schema scripts/schema.yaml --debug
+
+  insights:
+    docker:
+      - image: quay.io/reactiveops/ci-images:v11-stretch
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 18.09.3
+      - run:
+          name: Run Insights Checks
+          command: |
+            curl https://insights.fairwinds.com/v0/insights-ci.sh | bash
```

Comment:
We should probably go ahead and do the -L

Reply:
🤦